engine/src/test/java/org/hibernate/search/testsupport/TestConstants.java

```diff
@@ -89,4 +85,11 @@ public static String getIndexDirectory(Class<?> testClass) {
 	public static boolean arePerformanceTestsEnabled() {
 		return Boolean.getBoolean( "org.hibernate.search.enable_performance_tests" );
 	}
+
+	private static class CallerProvider extends SecurityManager {
+
+		public Class<?> getCallerClass() {
+			return getClassContext()[2];
```

Comment:
The static initialization will happen only once per JVM.
Does this imply that our testsuite now requires surefire to run with "fork" ?